### PR TITLE
Add https server document

### PR DESCRIPTION
**Pull request by ganmacs**
ref: https://github.com/fluent/fluentd/pull/2787

### CODE REVIEW

File: developer/api-plugin-helper-http_server.md [hunk 1]
```diff
@@ -38,13 +38,32 @@ in plugin's `stop` method. The plugin stops launched http server automatically.
...
+
+- `title`: the name of listen thread. this name must be unique
+- `addr`: Adderess to listen to
+- `port`: Port to listen to.
+- `logger`: Logger which is used in server helper
+- `default_app`: Use this object when server received a request whose path is not registered. This object must have `#call` or be a `Proc` object.
+- `proto`: protocol type. `:tcp`, `:tls`.  default value is tcp
+- `tls_opts`: options for TLS. same as the [Server Helper's  server_create_connection tls_options](/developer/api-plugin-helper-server)
+
+### http\_server\_create\_https\_server(title, addr:, port:, logger:, default\_app: nil, &block)
```
Comment (repeatedly) [inline developer/api-plugin-helper-http_server.md]: `tls_opts` is missing.

Comment (ganmacs) [reply developer/api-plugin-helper-http_server.md]: :bow: https://github.com/fluent/fluentd-docs-gitbook/pull/144/files#diff-49f6dfdd23dacafff7a525c45162ed59R43

**Comment by ganmacs** [comment]
May I merged this? @repeatedly

**Comment by repeatedly** [comment]
Released v1.9.1. Merged.

**Pull request merged**